scripts/deploy/09_eth_put_theta_vault.ts

```diff
@@ -0,0 +1,163 @@
+import { run } from "hardhat";
```

Comment:
One note, we will not be releasing an USDC-collateralized ETH Put vault, so we should:
- Rename this script to just be AVAX put
- Return early when `chainId != AVAX`

Reply:
Done